src/main/java/gift/config/KakaoProperties.java

```diff
@@ -0,0 +1,55 @@
+package gift.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
```

Comment:
오 `@ConfigurationProperties`를 활용해주셨군요! 💯 
해당 `@Configuration`은 빼주셔도될 것 같습니다ㅎㅎ 해당 애노테이션이 [어떤 용도](https://mangkyu.tistory.com/75)로 사용되는지 학습해보시면 좋을 것 같습니다ㅎㅎ
```suggestion
@Configuration
```

Reply:
@Configuration 을 제거하고 @ConfigurationPropertiesScan("gift.config") 을 이용하는걸로 바꿨습니다! 감사합니다.

Reply:
ConfigurationPropertiesScan 적용 너무 좋은 것 같습니다! 💯